src/reactive_state_access.rs

```diff
@@ -646,12 +646,28 @@ mod test {
     #[test]
     fn test_update() {
         a_reversible().update(|state| *state = 45);
-        assert_eq!(a_reversible().get(), 45, "We should get 3 as value for a");
+        assert_eq!(a_reversible().get(), 45, "We should get 45 as value for a");
+
+        a().update(|state| *state = 40);
+        assert_eq!(a().get(), 40, "We should get 40 as value for a");
     }
 
-    // #[test]
-    // fn test_update() {
-    //     a_reversible().update(|state| *state = 45);
-    //     assert_eq!(a_reversible().get(), 45, "We should get 3 as value for a");
-    // }
+    #[test]
+    fn test_delete() {
```

Comment:
This test is failing.

As a user I am expecting removing the state, so we should have no state after the deleting.

Is this right ?